doc/nrf/protocols/matter/getting_started/low_power_configuration.rst

```diff
@@ -118,6 +118,11 @@ To configure the LIT, CIP, UAT or DSLS, use the following Kconfig options:
   The DSLS support allows the application to dynamically switch between SIT and LIT modes, as long as the requirements for these modes are met.
   This option is by default disabled for the LIT device.
 
+You can enable optional reporting on entering the active mode.
+When enabled, the device sends a data report to the subscribed devices once entered the active mode.
```

Comment:
```suggestion
When enabled, the device sends a data report to the subscribed devices.
```
I'd say you don't need to repeat the active mode, as it is mentioned just above.